### PR TITLE
Fixed Style of mailchimp form

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
- Fixes https://github.com/mitodl/micromasters/issues/1153

#### What's this PR do?
- Fixed Style of mailchimp form in dialog

#### Where should the reviewer start?
- home_page.html/ mailchimp dialog

#### How should this be manually tested?
- see home_page.html/ mailchimp dialog

@pdpinch @aliceriot 

#### Screenshots (if appropriate)
screen shot 2016-09-29 at 5 43 50 pm

### CODE REVIEW

**Comment by pdpinch** [comment]
@roberthouse54 can you take a quick look at this?

I don't think it's worth sweating over too much, as I'm still waiting on a new form. But if the styling looks good, I'd like to merge it.

**Comment by roberthouse54** [comment]
Seems you are using different styles/classes for buttons, form fields, and also the modal header. 

Buttons: Please use the same button styles that we have elsewhere (Example: look on the settings page). 

Form Fields: Use the form fields styles that we have elsewhere (example: in the Sign Up). 

Modal Header: Use the header style for the modal that we use on the profile page... clicking "Add a degree, etc).

**Comment by amir-qayyum-khan** [comment]
@roberthouse54 these  ``Sign Up`` fields and model are part if react-mdl lib.
They dont have custom scss. Though found classes for buttons
@pdpinch updated text.

screen shot 2016-10-03 at 4 05 34 pm

**Comment by roberthouse54** [comment]
Can you check with another engineer on this... These are different form fields than we are using in other places, and I just want consistency.

**Comment by aliceriot** [comment]
The form fields are different than what we're using elsewhere. Unfortunately, though, the form fields we're using elsewhere are implemented in JavaScript (they're party of [material-ui](http://www.material-ui.com/)) so we can't use them here directly. Are we ok with what they look like? If not, we could try to get the standard material-design-light text fields working in plain HTML: https://getmdl.io/components/index.html#textfields-section

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -228,72 +228,58 @@ <h2 class="quote">"Micromasters is an important project for me. I believe in the
...
+                    <label><input type="radio" name="MMERGE2" value="I am an interested learner" id="learner">
+                        I am an interested learner
+                    </label>
+                  </div>
+                  <div class="radio">
+                    <label><input type="radio" name="MMERGE2" value="I represent an interested corporation" id="corporation">
+                        I represent an interested corporation
+                    </label>
+                  </div>
+
```
Comment (aliceriot) [inline cms/templates/cms/home_page.html]: could you delete this blank line?

Comment (amir-qayyum-khan) [inline cms/templates/cms/home_page.html]: fixed this

**Comment by amir-qayyum-khan** [comment]
@aliceriot do i need to import css from the link that you sent?
I think this style is implement from link as well as javascript code

**Comment by Ferdi** [comment]
I suggest we leave the markup that Amir has currently in the PR, even if it's different from what we use elsewhere.

**Comment by pdpinch** [comment]
@aliceriot if you accept Amir's latest change, I think we can merge this.

**Comment by aliceriot** [comment]
My comment above is more of a question, how much do we care that the mailchimp form doesn't really look like the rest of the site that much? If we think it's fine then this PR is ready to merge, if not it needs to be changed. @roberthouse54 ?

**Comment by roberthouse54** [comment]
Let's just merge it

**Pull request merged**